docs/conf.py

```diff
@@ -53,7 +53,7 @@
 # Functions that this calls are below
 def setup(app: Sphinx):
     app.connect("builder-inited", update_contributing)
-
+    app.add_crossref_type("team", "team")
```

Comment:
So cool. :)

Reply:
I just discovered this yesterday and am going to over use the shit out of it for a while I suspect haha